src/Sulu/Bundle/SecurityBundle/Resources/js/containers/Permissions/types.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+export type ContextPermission = {
+    context: string,
+    id: ?number,
+    permissions: {[string]: boolean},
+};
+
```

Comment:
No double new line. Seems like the behavior of eslint changed here in some version, so we also need to set the `maxEOF` option to `0` in the [`no-multiple-empty-lines`](https://eslint.org/docs/rules/no-multiple-empty-lines) rule.